### DISCUSSION TITLE
Update to French locale

### DISCUSSION THREAD

**Discussion by GM43299** [aclap-dev/video-downloadhelper#1007] [Language & Translation]
Hello,
I just filled in the missing translations for the French locale. Hopefully that helps...
[fr.txt](https://github.com/user-attachments/files/16203409/fr.txt)

**Comment by paulrouget** [comment]
Sorry we already got a translation for that language 🙏

I recommend that you try the upcoming version (v9.1 which should be released next week) see if the translation is correct.

If you want to keep track of the current translation status of your language, I recommend following this github repository: https://github.com/aclap-dev/vdhlocales
